handlers/validator.go

```diff
@@ -544,7 +544,8 @@ func Validator(w http.ResponseWriter, r *http.Request) {
 	})
 
 	g.Go(func() error {
-		if validatorPageData.ActivationEpoch > 100_000_000 {
+		// we only need to get the queue information if the action epoch is in the future and we have an eligibility epoch
```

Comment:
**nitpick:** The first part of the comment could be confusing. I would write it as something like
"// we only need to get the queue information if we don't have an activation epoch but we have an eligibility epoch"

Reply:
yes yours sounds better